bin/xud

```diff
@@ -9,6 +9,10 @@ const { argv } = require('yargs')
       type: 'number',
       alias: 'r',
     },
+    'rpc.defaultOrderAmount': {
+      describe: 'How many orders to return by default',
+      type: 'number',
+    },
```

Comment:
I am not sure. Third party applications and exchanges would use some kind of default amount anyway. If they specify it every time they request the orders or can configure it in the config of `xud` doesn't really matter. 

 But if minor details like this get added to the config it could get cluttered quickly.

Reply:
@sangaman any thought on that?